spyder/app/mainwindow.py

```diff
@@ -88,77 +83,44 @@
 from qtawesome.iconic_font import FontError
 
 
-#==============================================================================
+# ============================================================================
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
 # be set before creating the application.
-#==============================================================================
+# ============================================================================
 from spyder.config.manager import CONF
 
-if hasattr(Qt, 'AA_EnableHighDpiScaling'):
-    QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling,
-                                  CONF.get('main', 'high_dpi_scaling'))
 
-#==============================================================================
+# ============================================================================
 # Get CLI options and set OpenGL backend. This attibute must
 # be set before creating the application. See spyder-ide/spyder#11227
-#==============================================================================
+# ============================================================================
 from spyder.app.utils import set_opengl_implementation
 from spyder.app.cli_options import get_options
 from spyder.config.base import running_under_pytest

```

Comment:
Please move this import to a better place.

Reply:
Ok